recipes/localhgt/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.0.5" %}
+
+package:
+  name: localhgt
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepomicslab/LocalHGT/archive/refs/tags/v{{ version }}.tar.gz
+  md5: 9a082c8f5403b0b6a7003f7d825ce8cf
+  
+
+build: 
+  number: 0
+  entry_points: 
+    - localhgt = localhgt:main
+  noarch: generic
```

Comment:
Remove this line. You need C/C++ compilers, so it is not noarch. Maybe this is the reason why $CXX is not exported.
See https://dev.azure.com/bioconda/25074ded-3133-43ca-af80-895ab87b53cb/_apis/build/builds/55545/logs/26 for the full logs of https://github.com/bioconda/bioconda-recipes/pull/47056/files. This is how it should work for your recipe too.

Reply:
Thanks. However, after I deleted this section, it reports 
```
ERROR: recipes/localhgt/meta.yaml:19: should_be_noarch_generic: The recipe should be build as `noarch`
```